src/Arcus.Scripting.Storage.Table/Scripts/Set-AzTableStorageEntities.ps1

```diff
@@ -0,0 +1,71 @@
+param(
+    [Parameter(Mandatory = $true)][string] $ResourceGroupName = $(throw "Name of resource group is required"),
+    [Parameter(Mandatory = $true)][string] $StorageAccountName = $(throw "Name of Azure storage account is required"),
+    [Parameter(Mandatory = $true)][string] $TableName = $(throw "Name of Azure table is required"),
+    [Parameter(Mandatory = $true)][string] $ConfigurationFile = $(throw "Path to the configuration file is required")
+)
+
+if ((Test-Path -Path $ConfigurationFile) -eq $false) {
```

Comment:
Now I'm nitpicking 😅 , we could have used `-not (Test-Path -Path $ConfigurationFile)` here, but it's okay 😉 .

Reply:
Changed in [`8b49c27` (#222)](https://github.com/arcus-azure/arcus.scripting/pull/222/commits/8b49c277670f001e1a32c95f4dae953c835f41a0) 😄